EIPS/eip-NaN.md

```diff
@@ -0,0 +1,81 @@
+---
+eip: .nan
```

Comment:
```suggestion
eip: 6958
```
Issuing EIP number 6958 as this is the PR number.  Please also update the filename